tests/e2e/test_pv_scale_ocs_create_delete_pvcs.py

```diff
@@ -0,0 +1,338 @@
+                                    namespace=self.namespace))
+                log.info(f"Created pod {pod_name} using pvc {pvc_obj.name}")
+            for pvc_obj in cephfs_rwx_pvcs:
+                pod_name = helpers.create_unique_resource_name(
+                    resource_description=f"{pvc_obj.name}-pod",
+                    resource_type="pod")
+                future_temp_pod_objs.append(
+                    executor.submit(helpers.create_pod,
+                                    interface_type=constants.CEPHFS_INTERFACE,
+                                    pvc_name=pvc_obj.name, do_reload=False,
+                                    namespace=self.namespace))
+                log.info(f"Created pod {pod_name} using pvc {pvc_obj.name}")
+            for pvc_obj in rbd_rwo_pvcs:
+                pod_name = helpers.create_unique_resource_name(
+                    resource_description=f"{pvc_obj.name}-pod",
+                    resource_type="pod")
+                future_temp_pod_objs.append(
+                    executor.submit(helpers.create_pod,
+                                    interface_type=constants.CEPHBLOCKPOOL,
+                                    pvc_name=pvc_obj.name, do_reload=False,
+                                    namespace=self.namespace))
+                log.info(f"Created pod {pod_name} using pvc {pvc_obj.name}")
+            # TODO RWX PVC pod Creation
+            # for pvc_obj in rbd_rwx_pvcs:
+            #     pod_name = helpers.create_unique_resource_name(
+            #         resource_description=f"{pvc_obj.name}-pod",
+            #         resource_type="pod")
+            #     future_temp_pod_objs.append(
+            #         executor.submit(helpers.create_pod,
+            #                         interface_type=constants.CEPHBLOCKPOOL,
+            #                         pvc_name=pvc_obj.name, do_reload=False,
+            #                         namespace=self.namespace))
+            #     log.info(f"Created pod {pod_name} using pvc {pvc_obj.name}")
+
+        temp_pod_objs = [pod_obj.result() for pod_obj in future_temp_pod_objs]
+        log.info(f"Create {len(temp_pod_objs)} pods")
+        self.all_pod_obj.extend(temp_pod_objs)
+        # Check for all the pods are in Running state, in above pod creation
+        # Not waiting for the pod to be created because of threads usage
+        with ThreadPoolExecutor() as executor:
+            for pod in temp_pod_objs:
+                future_pod_objs.append(
+                    executor.submit(helpers.wait_for_resource_state,
+                                    pod, 'Running', timeout=self.timeout))
+        check_objs = [obj.result() for obj in future_pod_objs]
+        # If pods not up raising exception/failure
+        if False in check_objs:
+            raise TimeoutExpiredError
+        # Added run IO part for TC 'OCS-679'
+        if self.test_case == 'ocs679':
```

Comment:
not good.
The test should not be dependent on the test case ID

Reply:
removed ,, will handle it part of pytest.parametrized